VisualStudio/oqs/dll.def

```diff
@@ -1,19 +1,33 @@
 LIBRARY oqs
 EXPORTS
-	OQS_randombytes_switch_algorithm
-	OQS_randombytes_custom_algorithm
-	OQS_randombytes
-	OQS_randombytes_system
-	OQS_KEM_new
-	OQS_KEM_keypair
-	OQS_KEM_encaps
-	OQS_KEM_decaps
-	OQS_KEM_free
-	OQS_SIG_new
-	OQS_SIG_keygen
-	OQS_SIG_sign
-	OQS_SIG_verify
-	OQS_SIG_free
-	OQS_MEM_cleanse
-	OQS_MEM_secure_free
-	OQS_MEM_insecure_free
+	OQS_MEM_cleanse
+	OQS_MEM_secure_free
+	OQS_MEM_insecure_free
+	OQS_randombytes_switch_algorithm
+	OQS_randombytes_custom_algorithm
+	OQS_randombytes
+	OQS_randombytes_nist_kat_init
+	OQS_KEM_frodokem_640_aes_keypair

```

Comment:
Why are the FrodoKEM functions listed explicitly here, but no others?

Reply:
Only Frodo and qTESLA have OQS_API-exported function, which is what I used to populate the DLL def file (but qTESLA isn't supported on Windows).

Reply:
Yes indeed; we should open a work item to do this on the Linux build, and I can update this PR once the OQS_API tag is properly set everywhere.